sentinelsat/sentinel.py

```diff
@@ -401,7 +401,7 @@ def download(self, id, directory_path='.', checksum=False):
         checksum : bool, optional
             If True, verify the downloaded file's integrity by checking its MD5 checksum.
             Throws InvalidChecksumError if the checksum does not match.
-            Defaults to False.
+            Defaults to True.
```

Comment:
Why do we write this at all? The default is in the function declaration in this case (would be different if the default was `None`).

Reply:
You are correct. I agree that there's little benefit to having a line in the method's doc specifying what the default value is, when it's defined in the method's signature. However, I did not write the original documentation -- I merely updated it.

Reply:
Documenting the default is optional according to the [numpy docstring standard](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#id5) that we use.

> Optional keyword parameters have default values, which are displayed as part of the function signature. They can also be detailed in the description:

We can remove it. In the case of this PR I'd say updating is the preferred choice. If we want to remove the documented defaults we should do it consistently throughout the codebase.